x-pack/plugins/observability/public/pages/slo_edit/components/common/data_preview_chart.tsx

```diff
@@ -189,7 +239,40 @@ export function DataPreviewChart({
           )}
           {isSuccess && (
             <Chart size={{ height: 160, width: '100%' }}>
-              <Tooltip type="vertical" />
+              <Tooltip
+                type="vertical"
+                body={({ items }) => {
+                  const firstItem = items[0];
```

Comment:
This section is for adding this tooltip, right?

<img src="https://github.com/elastic/kibana/assets/12370520/7e61973b-acf6-444d-b53a-5b37062d096b" width=200 />

How do you know we will have `items[0]`? Is it something that chart checks, or are we doing extra checks ourselves? I didn't fully understand where this items come from 🤔

Reply:
yeah this is essentially chart inner tooltip , if there won't be any items, this won't be called/rendered at all.